files/elevatezoom/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "elevatezoom",
+  "repo": "elevateweb/elevatezoom",
+  "files": {
+    "include": ["jquery.elevatezoom.js"]
```

Comment:
This file isn't minified and the files they provide are don't follow naming conventions. Mind removing the update.json and instead including the `jquery.elevateZoom-3.0.8.min.js` file (under `3.0.8/jqueryElevateZoom.js` or similar)